assets/js/script.js

```diff
@@ -0,0 +1,195 @@
+function openNav() {
+  document.getElementById('myNav').style.width = '100%';
+}
+
+function closeNav() {
+  document.getElementById('myNav').style.width = '0%';
+}
+
+const closeBtn = document.getElementById('close-btn');
+const IconMenu = document.getElementById('click-image');
+const hamItem = document.getElementsByClassName('nav-list');
+
+closeBtn.addEventListener('click', () => {
+  closeNav();
+});
+IconMenu.addEventListener('click', () => {
+  openNav();
+});
+
+for (let index = 0; index < hamItem.length; index += 1) {
+  hamItem[index].addEventListener('click', () => {
+    closeNav();
+  });
+}
+
```

Comment:
All your images are broken kindly work on it:
![Screen Shot 2021-10-14 at 15 07 18](https://user-images.githubusercontent.com/22258274/137388092-677db3cb-f478-4b4d-9f1b-2885fc5b6c92.png)